pkg/auth/providers/common/ldap/ldap_util.go

```diff
@@ -97,6 +101,7 @@ func IsType(search []*ldapv2.EntryAttribute, varType string) bool {
 		if attrib.Name == "objectClass" {
 			for _, val := range attrib.Values {
 				if strings.EqualFold(val, varType) {
+					logrus.Debugf("Object is type: %s", varType)
```

Comment:
might need some more context in this log

Reply:
sure a bit, let me add